api/controllers/swagger.go

```diff
@@ -235,6 +235,16 @@ func (s *SwaggerController) ServeSwaggerSpec(c *fireball.Context) (fireball.Resp
 				},
 			},
 			"/service": {
+				"get": {
```

Comment:
`POST /service` for Create Service needs to be included in this entity block (based on the routes `api/controllers/service.go`)

Reply:
It will be once Create Service exists